invoice.py

```diff
@@ -449,16 +496,21 @@ def _sign_facturae(self, xml_string, certificate_password):
         unsigned_file.close()
 
         cert_file = NamedTemporaryFile(suffix='.pfx', delete=False)
-        cert_file.write(self.company.facturae_certificate)
+        cert_file.write(certificate_facturae)
```

Comment:
Temporary files should be removed at the end. The best option is to use a `with` statement.